docs/wiki/The CI environment - Pipeline usage.md

```diff
@@ -0,0 +1,141 @@
+
+## Add a new module pipeline
+
+To add a new module pipeline we recommend to create a copy of a currently existing module pipeline and adjust all module-specific properties. The registration of the pipeline depends on the [DevOps platform](#devops-tool-specific-guidance) you're using.
+# Operate the dependency pipeline
+
+The dependency pipeline must be triggered manually and deploys a set of resources we reference as part of our module tests (e.g. Virtual Networks, Log Analytics Workspace, Key Vaults).
+
+Triggering the pipeline is as easy as navigating to it in your corresponding DevOps tool and running the pipeline.
+
+> **Note:** While operating the dependency pipeline is simple, make sure to set it up correctly as described in the [Getting Started - Dependency pipeline](./Getting%20started%20-%20Dependency%20pipeline) section.
+
+## Add a new dependency
+
+Depending on what you want to test in your module pipeline, you may want to include additional resources to your dependency pipeline. If so, make sure to include an additional parameter file for each service you require under `utilities/pipelines/dependencies`. Once done, you need to add the deployment to the pipeline itself making sure to deploy the new resources in the correct order. The implementation depends on the [DevOps platform](#devops-tool-specific-guidance) you're using.
+
+---
+
+# DevOps-Tool-specific guidance
+
+This section provides a step-by-step guideline on how to operate the pipelines based on the specific chosen DevOps platform, GitHub or Azure DevOps.
+
+## GitHub workflows
+
+This section focuses on _GitHub_ Actions & Workflows.
+
+<details>
+<summary>GitHub workflows</summary>
+
+  ### Trigger a workflow
+
+  To trigger a workflow in _GitHub_:
+
+  1. Navigate to the 'Actions' tab in your repository.
+
+     <img src="./media/ghActionsTab.png" alt="Actions tab" height="100">
+
+  1. Select the pipeline of your choice from the list on the left, followed by 'Run workflow' to the right. You can then select the branch of your choice and confirm the execution by clicking on the green 'Run workflow' button.
+
+     <img src="./media/gHtriggerPipeline.png" alt="Run workflow" height="350">
+
+  >**Note**: Depending on the pipeline you selected you may have additional input parameters you can provide aside from the branch. An outline can be found in the [Module pipeline inputs](./The%20CI%20environment%20-%20Pipeline%20design#module#module-pipeline-inputs) section.
+
+  ### Register a workflow
+
+  To register a workflow in _GitHub_ you have to create the workflow file (`.yml`) and store it inside the folder `.github/workflows`.
+  > ***Note:*** Once merged to [main\|master], GitHub will automatically list the new workflow in the 'Actions' tab.
```

Comment:
```suggestion
  > ***Note:*** Once merged to [main\|master], GitHub will automatically list the new workflow in the 'Actions' tab. Workflows are not registered from a branch unless you specify a temporal push trigger.
```

Reply:
Would this be enough to cover the previous open comment?